xamarin/Xamarin.Forms
# [Shell,Android] Allow transparent background on Shell Header

### Description of Change ###

Allow transparent Shell Header.

### Issues Resolved ### 

- fixes #13305 
- fixes #12309

### API Changes ###
 
 None

### Platforms Affected ### 

- Android

### Behavioral/Visual Changes ###

Header should be transparent and not white by default

### Before/After Screenshots ### 

Before
![image](https://user-images.githubusercontent.com/1235097/103895833-ec2dee80-50e8-11eb-99df-e9c44d949a42.png)

After
![image](https://user-images.githubusercontent.com/1235097/103895661-a96c1680-50e8-11eb-83c8-525ac6d1a663.png)

Not applicable

### Testing Procedure ###

- Go to the Shell FlyoutBackgroundTests issue page
- Click toggle Footer/header transparent 
- Make sure image is below the header

### PR Checklist ###
